### DISCUSSION TITLE
ポイント機能

### DISCUSSION THREAD

**Discussion by shigurenimo** [aipictors/aipictors#67] [アイデア - Ideas]
ユーザにポイントを付与して、全ての機能をポイント消費で制限するのはどうか？

**Comment by nakatsuka-k** [comment]
法的な問題を解決する必要がありますが
ユーザビリティの向上と、都度必要に応じて追加で
サービスをご利用されるユーザにとっては快適になると思います。
現在のサブスクについてはお得な量でポイント付与に
特典を切り替えてもいいかもしれません。
